test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -1294,6 +1294,16 @@ def forward(self, input1, input2, input3):
 
         self.run_test(TraceModel(), (x1, x2, x3), atol=10e-5)
 
+    def test_numpy_T(self):
+        class NumpyTranspose(torch.nn.Module):
+            def forward(self, x):
+                return x.T
+
+        x1 = torch.randn(4, 7)
+        self.run_test(NumpyTranspose(), x1)
+        x2 = torch.randn(4)
+        self.run_test(NumpyTranspose(), x2)
```

Comment:
I would test a scalar e.g. `torch.tensor(42)` as well

Reply:
Done. I actually removed the 1D test case, since as per [1.11 Release Notes](https://github.com/pytorch/pytorch/releases) using `.T` on anything that isn't 0D or 2D is deprecated.